code/game/objects/structures/fluff.dm

```diff
@@ -263,3 +271,221 @@
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
+	layer = 3.9
+	pass_flags = LETPASSTHROW
+	density = FALSE
+	anchored = TRUE
+	deconstructible = FALSE
+
+/obj/structure/fluff/books_sign
+	name = "crude sign"
+	desc = "A crudely-made sign with the words 'books' written."
+	icon = 'icons/misc/tourny_items.dmi'
+	icon_state = "books"
+	max_integrity = 9999
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
+	density = TRUE
+	anchored = TRUE
+	deconstructible = FALSE
+
+/obj/structure/fluff/hop_sign
+	name = "crude sign"
+	desc = "A crudely-made sign with the the symbol of a HoP's hat drawn."
+	icon = 'icons/misc/tourny_items.dmi'
+	icon_state = "paperwork"
+	max_integrity = 9999
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
+	density = TRUE
+	anchored = TRUE
+	deconstructible = FALSE
+
+/obj/structure/fluff/medieval_sign
+	name = "crude sign"
+	desc = "A crudely-made sign with the the symbol of a crown, implying that this is a medieval stand."
+	icon = 'icons/misc/tourny_items.dmi'
+	icon_state = "medieval"
+	max_integrity = 9999
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
+	density = TRUE
+	anchored = TRUE
+	deconstructible = FALSE
+
+/obj/structure/fluff/oranges_sign
+	name = "crude sign"
+	desc = "A crudely-made sign with the a drawn orange. Has the day of the juicer finally arrived?"
+	icon = 'icons/misc/tourny_items.dmi'
+	icon_state = "juicer"
+	max_integrity = 9999
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
+	density = TRUE
+	anchored = TRUE
+	deconstructible = FALSE
+
+/obj/structure/fluff/ruinous_sign
+	name = "crude sign"
+	desc = "A crudely-made sign with an even more crude symbol. Who made this crap? A bunch of evil lunatics?"
+	icon = 'icons/misc/tourny_items.dmi'
+	icon_state = "evil"
+	max_integrity = 9999
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
+	density = TRUE
+	anchored = TRUE
+	deconstructible = FALSE
+
+/obj/structure/fluff/store_sign
+	name = "crude sign"
+	desc = "A crudely-made sign with two letters representing an ancient company. It's also the only company whose plastic is still expensive in 2560."
+	icon = 'icons/misc/tourny_items.dmi'
+	icon_state = "satire"
+	max_integrity = 9999
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
+	density = TRUE
+	anchored = TRUE
+	deconstructible = FALSE
+
+/obj/structure/fluff/headstone
+	name = "headstone"
+	desc = "R.I.P"
+	icon = 'icons/misc/tourny_items.dmi'
+	icon_state = "headstone"
+	max_integrity = 9999
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
+	density = TRUE
+	anchored = TRUE
+	deconstructible = FALSE
+
+/obj/structure/fluff/thefuckinggate // this shit is 9 fucking tiles, dude. This is a bruh moment.
+	name = "Gate of The Toolbox Tournament"
```

Comment:
should "the" be capitalized here? is it a part of the toolbox tournament's name?